docs/ethics/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: layouts/ethics.njk
+title: Designing ethical services
+---
+
+Anyone designing and building MoJ services must be mindful of the impact of their design on people, society and the planet.
+
+1. Learn and share the [MoJ ethical design principles](/ethics/principles).
+2. [Follow our ethics standards](/ethics/standards) and make ethics a practical part of your design process.
```

Comment:
(Speaking as a reader) it would be good to explain what the difference is between principles and standards

I expect you're also going to get some questions about who wrote these, who maintains them, and what authority they come with, so it might be good to lay some of this out here.

Reply:
Really good points. I'll let the ethics team know about the suggestions so it can be considered within the testing they're going to do.